### PR TITLE
1.17 pipe to furnance fix

### PR DESCRIPTION
**Pull request by colaman90**
Added new 1.17 items to the smeltable list

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Sat Jul 31 23:05:01 CEST 2021
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```
Comment (me4502) [inline gradle/wrapper/gradle-wrapper.properties:2]: Please don't change the gradle distribution here

File: src/main/resources/plugin.yml [hunk 2]
```diff
@@ -3,5 +3,5 @@ main: com.sk89q.craftbook.bukkit.CraftBookPlugin
 version: "${internalVersion}"
 dev-url: http://dev.bukkit.org/server-mods/craftbook/
 softdepend: [WorldEdit, WorldGuard, ProtocolLib, Vault]
-api-version: 1.13
+api-version: 1.17
```
Comment (me4502) [inline src/main/resources/plugin.yml:6]: Please don't change the API version, we use WorldEdit's compat system

**Review by me4502 — CHANGES_REQUESTED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (colaman90) [reply gradle/wrapper/gradle-wrapper.properties]: Revert to old Settings

File: src/main/resources/plugin.yml (hunk 2 above)
Comment (colaman90) [reply src/main/resources/plugin.yml]: Revert to old Settings

**Comment by bmack94** [comment]
Will this be merged? I have a 1.17 server running and would like to use the pipe for the new items

**Review by me4502 — APPROVED** [review]

**Pull request merged**